kuma/users/views.py

```diff
@@ -247,6 +279,55 @@ def ban_user_and_cleanup_summary(request, user_id):
                   context)
 
 
+def revision_by_distinct_doc(list_of_revisions):
+    documents = {}
+    for rev in list_of_revisions:
+        documents.setdefault(rev.document_id, rev)
+        if documents[rev.document_id].id < rev.id:
+            documents[rev.document_id] = rev
+
+    return [documents[doc_id] for doc_id in sorted(documents)]
+
```

Comment:
I like avoiding the extra SQL calls, and this appears like it will work. Since queries are already sorted by ``-id``, the first revision for a document is always the highest ID, so the ``if documents[rev.document_id].id < rev.id`` branch is [never taken](https://codecov.io/gh/mozilla/kuma/src/1203d19f6d3fe1b66e609a1e62b74525c7816744/kuma/users/views.py#L286).  I don't think we should replace it with an assert though, so no change.